examples/index.json

```diff
@@ -211,7 +211,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Errors",
+                "name": "Error reporting",
```

Comment:
Why are we changing these? I wouldn't change the titles to these.

Reply:
Changed to be consistent with the URL. Reverted the name back to `Errors`.